CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## [v0.8.2](https://github.com/lystable/dennis/tree/v0.8.2) (Unreleased)
+
+[Compare to previous release](https://github.com/lystable/dennis/compare/v0.8.1...v0.8.2)
+
+**Added**:
+
+- Release v0.8.1 
```

Comment:
This changelog looks incorrect. We should keep it up to date even when dev-ing on the tool